src/components/portrait/portrait.test.tsx

```diff
@@ -0,0 +1,197 @@
+import React from "react";
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import MD5 from "crypto-js/md5";
+import Portrait from ".";
+import Profile from "../profile";
+import { testStyledSystemMargin } from "../../__spec_helper__/__internal__/test-utils";
+import CarbonProvider from "../carbon-provider";
+
+testStyledSystemMargin((props) => <Portrait {...props} />);
+
+test("renders with a default individual icon", () => {
+  render(<Portrait />);
+
+  const icon = screen.getByTestId("icon");
+  expect(icon).toBeVisible();
+  expect(icon).toHaveAttribute("type", "individual");
+});
+
+test("renders with a custom icon, if a valid icon is provided via the `iconType` prop", () => {
+  render(<Portrait iconType="bin" />);
+
+  const icon = screen.getByTestId("icon");
+  expect(icon).toBeVisible();
+  expect(icon).toHaveAttribute("type", "bin");
+});
+
+test("renders with initials, if the `initials` prop is provided", () => {
+  render(<Portrait initials="NM" />);
+
+  expect(screen.getByText("NM")).toBeVisible();
+});
+
+test("if the `Portrait` component is rendered within `Profile` which has a value for the `name` prop, `Portrait` can render with initials generated from said `name` prop", () => {
+  render(<Profile name="Neil McCauley" />);
```

Comment:
**question**: just wondering, what are your thoughts on placing the `Profile` related tests within `Profile`'s test file? It might help improve overall organisation.

Reply:
Yeah I wouldn't be opposed to that. I'll add an enzyme test for now, when I do the conversion for `Profile`, i'll re-include these tests

Reply:
After looking, it appears these are actually already sufficently covered in `Profile.spec`, name -> initials is on line 84, and name prop -> alt tag is on line 404